redisinsight/ui/src/components/new-index/selection-box/SelectionBox.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { BoxSelectionGroup, BoxSelectionGroupBox } from '@redis-ui/components'
+import styled from 'styled-components'
+import { Text, Title } from 'uiSrc/components/base/text'
+
+export interface BoxSelectionOption<T extends string = string>
+  extends BoxSelectionGroupBox<T> {
+  text?: string
+}
+
+const StyledBoxContent = styled.div`
+  padding: 16px;
+  text-align: left;
+`
+
+const StyledTitle = styled(Title)`
+  margin-top: 4px;
+`
+
+const StyledText = styled(Text)`
+  margin-top: 4px;
+`
+
+const StyledDisabledBar = styled.div`
+  padding: 2px 0;
+  /* Design color: #f4f5f6 */
+  background: var(--comboBoxBadgeBgColor);
+  /* Design color: #5c707a */
+  color: var(--badgeIconColor);
+  border-radius: 2px;
+  border-bottom: 1px solid var(--controlsBoxShadowColor);
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+`
+
+const SelectionBox = ({ label, text, disabled, ...rest }: any) => (
```

Comment:
Why `any`, don't we have any type/interface provided by Redis UI that we can rely upon?